recipes/libnvimgcodec/build.sh

```diff
@@ -0,0 +1,66 @@
+#! bash
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+mkdir build
+cd build
+
+export CUDAARCHS="50"
```

Comment:
Just to make sure I'm understanding -- is this the oldest supported arch and we use that to generate PTX?

Reply:
No! This is a good catch! It's for debugging purposes so that the build completes faster. I need to remove or replace this so code for more archs is generated.

Reply:
```suggestion
```